src/components/Identity/index.js

```diff
@@ -62,7 +62,7 @@ const createIdentity = ({ config, logger, cookie }) => {
             logger.log("Delaying request while retrieving ECID from server.");
             promise = deferredForEcid.promise.then(() => {
               logger.log("Resuming previously delayed request.");
-              addIdsContext(payload, ecid);
+              addIdsContext(payload, getEcid());
```

Comment:
We had a bug where this line was referring to the `ecid` variable, which was `undefined` because we didn't have an ECID when that variable was declared. Instead, we need to pull the ECID anew now that we know an ECID has been retrieved. I wrote a functional test case in TestRails that should cover this.